editors/code/src/installation/language_server.ts

```diff
@@ -119,6 +128,14 @@ export async function ensureLanguageServerBinary(
     }
 
     function isBinaryAvailable(binaryPath: string) {
-        return spawnSync(binaryPath, ["--version"]).status === 0;
+        const res = spawnSync(binaryPath, ["--version"]);
+
+        // ACHTUNG! `res` type declaration is inherently wrong, see
+        // https://github.com/DefinitelyTyped/DefinitelyTyped/issues/42221
```

Comment:
Please take a not on this caveat
https://github.com/DefinitelyTyped/DefinitelyTyped/issues/42221